overlord/snapstate/snapstate.go

```diff
@@ -69,25 +69,35 @@ func isParallelInstallable(snapsup *SnapSetup) error {
 	return fmt.Errorf("cannot install snap of type %v as %q", snapsup.Type, snapsup.InstanceName())
 }
 
+func canInstallSnapdSnap(st *state.State) error {
+	model, err := Model(st)
+	if err != nil && err != state.ErrNoState {
+		return err
+	}
+	// any model that uses a base caninstall the snapd snap
+	if model != nil && model.Base() != "" {
```

Comment:
From irc: `- it seems we will transition automatically if base/model exists, regardless of the experimental flag, and the flag will only be used with other models. is that intended?`